docs/medical-api/more-info/contribution.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: "Contributing Data Back"
+description: "How to contribute data back to Health Information Exchange (HIE) networks"
+---
+
+## Importance and Requirements
+
+Contributing data back to Health Information Exchange (HIE) networks is a critical requirement for all our customers. This bidirectional flow of information ensures that the HIE ecosystem remains up-to-date and valuable for all participants.
+
+### Key Points:
+
+1. **Mandatory Requirement**: Data contribution back to the HIE networks is not optional. It's a fundamental obligation for all customers using our services.
+
+2. **Timeframe**: Data must be contributed back soon after a patient has an encounter with a provider and the data is ready to be made available. This ensures that the HIE network has timely access to the most recent patient information.
+
+3. **Consequences of Non-Compliance**: Failure to comply with the data contribution requirement may result in revocation of access to the HIE network. This could significantly impact your ability to access and utilize crucial health information.
+
+<Info>
+  There is an exception to contributing data back through Metriport which applies to customers that
+  are working with facilities that are already connected to the networks and contributing data back.
+  These would be registered as "OBO" or "On Behalf Of" facilities.
+</Info>
+
+## How to Share Data
+
+To facilitate the data contribution process, we provide two primary endpoints that customers can use to submit data back to the HIE networks:
+
+- **Create Patient Consolidated**
+
+  - Use this [endpoint](/medical-api/api-reference/fhir/create-patient-consolidated) to create or update consolidated patient records.
+
+- **Document Upload URL**
+  - This [endpoint](/medical-api/api-reference/document/upload-url) is used for uploading clinical documents related to patient encounters.
+
+## Example Resource to Share
+
+Customers can upload various types of FHIR R4 resources to contribute data back to the HIE networks. Here's an example of an Encounter resource:
+
+```json
+[
```

Comment:
let's include a full example bundle here, with the practitioner, and practitioner NPI

let's also include a some bullet points with some common tips like:
- resources must point to the Metriport patient ID the data is being upload for, in this example it's `subject.reference`
- etc